finvizfinance/screener/overview.py

```diff
@@ -24,16 +26,41 @@ class Overview:
 
     v_page = 111
 
-    def __init__(self):
+    def __init__(self, username=None, password=None):
```

Comment:
Login needs to be done before calling this module if user wants the elite. Also, add a parameter for choosing the elite.

Reply:
Added parameter in order to choose Elite mode.